pom.xml

```diff
@@ -23,17 +23,23 @@
         <encoding>UTF-8</encoding>
         <project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
         <project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
-        <restolino.version>0.6.0</restolino.version>
-        <fasterxml.jackson.version>2.15.0</fasterxml.jackson.version>
+        <restolino.version>0.7.1</restolino.version>
+        <fasterxml.jackson.version>2.16.1</fasterxml.jackson.version>
         <apache.poi.version>3.17</apache.poi.version>
-        <spring.version>5.3.20</spring.version>
-        <dp.logging.version>2.0.0-beta.8</dp.logging.version>
-        <batik.version>1.16</batik.version>
-        <logback.version>1.2.11</logback.version>
+        <spring.version>5.3.31</spring.version>
+        <dp.logging.version>v2.0.0-beta.11</dp.logging.version>
+        <batik.version>1.17</batik.version>
+        <logback.version>1.3.14</logback.version>
     </properties>
 
     <dependencyManagement>
         <dependencies>
+            <!-- Logging keep logging at the beginning here as the order of imports does matter to maven-->
```

Comment:
```suggestion
            <!-- Logging - keep logging at the beginning here as the order of imports does matter to maven-->
```

Reply:
Is there something in particular it's clashing with if you put it lower down?

Reply:
We found that dp-logging had a library missing _org.slf4j_ which required all java applications to add this dependency so that they would work. What we did is we added this dependency in dp-logging to be pulled in from there and avoid having to include it in every application. However, restolino is also using _org.slf4j_ but an older version of slf4j-api since we are only doing minor updates. By changing the order of dependencies and putting dp-logging above restolino the build then worked as it's using version 2.0.10 which is being pulled from dp-logging.

Reply:
Sure - so can you note this in the comment so we know exactly why it's being kept there and what we should avoid putting it below in future.